src/css/styles/_top.scss

```diff
@@ -0,0 +1,126 @@
+.top {
+  position: relative;
+}
+
+.nav {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: calc(100% - 2 * 2rem);
+  margin: 0 2rem;
+  font-size: 0.6rem;
+  border-bottom: solid $col-orange 3px;
+}
+
+.nav-logo {
+  float: left;
+  margin: 1rem 0;
+}
+
+.nav-menu {
+  float: right;
+}
+
+.nav-menu__list {
+  display: inline-block;
+  margin: 1rem;
+}
+
+.nav-menu__list__item {
+  display: inline;
+}
+
+.nav-link {
+  padding: 0.5rem;
+  color: $col-nav-link;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.nav-link:hover {
+  color: #fff;
+  text-decoration: underline;
+  transition: 0.2s ease-in-out;
+}
+
+.nav-menu__lang {
+  margin: 1rem;
+}
+
+// tablet and mobile
+
+.menu-toggle {
+  display: none;
+  float: right;
+  padding: 1rem 0;
+}
+
+.nav-mobile {
+  position: absolute;
+  top: 52px;
+  left: 100%;
+  z-index: 1000;
+  width: 100%;
+  height: calc(100% - 2 * 2rem);
+  padding: 2rem;
+  // vertical-align: middle;
+  background: $col-bg-gray-dark;
+  transition: 0.5s ease-in-out;
+}
+
+.nav-mobile.show {
+  left: 0;
+}
+
+.nav-link--mobile {
+  padding: 0;
+  font-size: 1.6rem;
+}
+
+.nav-mobile__list__item {
+  padding: 1rem 0;
+  border-bottom: 1px solid $col-banner-gray-dark;
+}
+
+.nav-mobile-kontakt {
+  display: none;
+  width: 100%;
+  margin-left: 0;
+}
+
+.nav-mobile-kontakt__item {
+  display: inline;
+}
+
+.nav-mobile-kontakt__link {
+  font-size: 0.8rem;
+}
```

Comment:
Mix of Polish and English in css and in imgs naming.

Reply:
Polish relates to specific areas named this way in the graphic layout we base on, English is used as a main language for the rest. Names in Polish remain in this language, for simplicity of reference- 1st Polish is the main language, 2nd- we dont have English translations yet (translations may vary from our naming convention, which would be confusing).
On the other hand I get the benefit from using just English. To be discussed.